resource/definitions/loader/3dsmax/camera-3dsmax-loader.json

```diff
@@ -0,0 +1,215 @@
+{
+  "type": "loader",
+  "name": "Camera Loader",
+  "asset_type": "cam",
+  "host_type": "3dsmax",
+  "ui_type": "qt",
+  "contexts": [
+    {
+      "name": "main",
+      "stages": [
+        {
+          "name": "context",
+          "plugins":[
+            {
+              "name": "context selector",
+              "plugin": "common_passthrough_loader_context",
+              "widget": "common_passthrough_loader_context"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "components": [
+    {
+      "name": "snapshot",
+      "file_formats": [".mb", ".ma"],
+      "stages": [
+        {
+          "name": "collector",
+          "plugins":[
+            {
+              "name": "Collect components from context",
+              "plugin": "common_context_loader_collector"
+            }
+          ]
+        },
+        {
+          "name": "importer",
+          "plugins":[
+            {
+              "name": "Import paths to Max",
+              "plugin": "max_native_loader_importer",
+              "options": {
+                "load_mode": "import",
+                "load_options": {
+                  "preserve_references": true,
+                  "add_namespace": true,
+                  "namespace_option": "file_name"
+                }
+              }
+            }
+          ]
+        },
+        {
+          "name": "post_importer",
+          "plugins":[
+            {
+              "name": "max",
+              "plugin": "common_passthrough_loader_post_importer"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "camera",
+      "file_formats": [".mb", ".ma"],
+      "stages": [
+        {
+          "name": "collector",
+          "plugins":[
+            {
+              "name": "Collect components from context",
+              "plugin": "common_context_loader_collector"
+            }
+          ]
+        },
+        {
+          "name": "importer",
+          "plugins":[
+            {
+              "name": "Import paths to Max",
+              "plugin": "max_native_loader_importer",
```

Comment:
This plugin is the same as the snapshot? so no specific plugin to export the camera here?

Reply:
This is loading